go/example/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+var host = flag.String("host", ":3423", "host address:port")
+
+type AgentInfo struct {
+	Owner  string
+	Taunts []string
```

Comment:
I imagine we'll want to watch replays of AI versus AI games. I'm willing to wager viewers will find them entertaining. It allows authors to add a personal touch. I bet it'll make it a smidge more fun to participate.